src/hooks/useHasInstalledApp.ts

```diff
@@ -0,0 +1,15 @@
+import {useOnyx} from 'react-native-onyx';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+const useHasInstalledApp = () => {
```

Comment:
Just a suggestion to make this even more explicit to what it is. 

```suggestion
const useHasLoggedIntoMobileApp = () => {
```